src/fn/fn-notification.scss

```diff
@@ -0,0 +1,92 @@
+@import "./fn-settings";
+
+$block: #{$fn-namespace}-notification;
+
+$fn-notification-padding: 1.5rem;
+$fn-notification-text-padding: 0.75rem;
+$fn-notification-border: 0.125rem solid;
+
+$notification-colors: (
+  "positive": ("color": $fn-color-green-500),
+  "critical": ("color":  $fn-color-orange-400),
+  "negative": ("color":  $fn-color-red-500),
+  "info": ("color":  $fn-color-blue-500)
+);
+
+.#{$block} {
+  @include fn-reset();
+  @include fn-flex(column);
+
+  padding: $fn-notification-padding;
+  border-radius: $fn-notification-text-padding;
+  background: $fn-color-base-white;
+  box-shadow: $fn-shadow-grey-level-2;
+  border: $fn-notification-border transparent;
+  width: 26.75rem;
+
+  &__main-container {
+    @include fn-reset();
+    @include fn-flex();
+
+    .#{$block}__text-container {
```

Comment:
Did not consider the specificity issue, I structured it like this because I figured text-container will always be a child of main container. But I will move this and use &__